jujugui/static/gui/src/app/components/budget-chart/_budget-chart.scss

```diff
@@ -38,7 +48,7 @@
         }
 
         &--existing {
-            background-color: #aee2b6;
+            background-color: $light-green;
```

Comment:
Personally I think its nicer to use the `lighten` function here to mod our $green/$success variables. For example: `lighten($success, 50%)`